docs/configuration.rst

```diff
@@ -14,9 +14,11 @@ as **1 minute**!
 
 Environment Variables
 ---------------------
-Most Node.js applications expect your confidential information to be
-exposed by the environment (not hard-coded in the application).  You
-should export your Stormpath information by running this in the shell:
+
+It is a best practice to store confidential information in environment
+variables (*don't hard-code it into your application*). You should store your
+confidential Stormpath information in environment variables.  You can do this
+by running the following commands in the shell::
```

Comment:
Nneed to remove one of the colons at the end of this line, to get the code block to render propertly.